src/current/v23.2/install-cockroachdb-windows.md

```diff
@@ -19,6 +19,10 @@ docs_area: deploy
 
 Use one of the options below to install CockroachDB.
 
+{{site.data.alerts.callout_success}}
+To connect to a CockroachDB cluster running any supported version of CockroachDB from your local system, you can download and install any available version of either the SQL Shell binary or the full binary for your operating system and architecture. Refer to [Releases](https://cockroachlabs.com/docs/releases/).
```

Comment:
Here I say "download and install any available version" because all the Windows downloads are experimental.